production.env

```diff
@@ -0,0 +1,2 @@
+BACKEND_URL    = https://shop.bootcamp.place/api/
+STRIPE_API_KEY = pk_test_51OhTCmDf0QjduFWJEpbxu9RBle3CZTnl7uDWLD68F2nu44nsLPEAtIawuGiqf4T6IJrK1o10Dnhy1FwsmP8dooms00HxIfUBe1
```

Comment:
**Security Issue**: Hardcoded sensitive keys in environment files can lead to security breaches if the codebase is exposed. <br> **Fix**: Remove hardcoded sensitive keys and use secure secrets management tools. <br> **Code Suggestion**: 
 ```
 -STRIPE_API_KEY = pk_test_51OhTCmDf0QjduFWJEpbxu9RBle3CZTnl7uDWLD68F2nu44nsLPEAtIawuGiqf4T6IJrK1o10Dnhy1FwsmP8dooms00HxIfUBe1
 +STRIPE_API_KEY = process.env.STRIPE_API_KEY
 ```